<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoLoadDialog</class>
 <widget class="QDialog" name="VideoLoadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>비디오 불러오기</string>
  </property>
  <widget class="QPushButton" name="LoadVideo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="text">
    <string>비디오 불러오기</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="title">
    <string>정보</string>
   </property>
   <widget class="QLabel" name="FileNameInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔바른고딕</family>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔바른고딕</family>
     </font>
    </property>
    <property name="text">
     <string>파일 이름</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="SaveVideo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="text">
    <string>비디오 저장하기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LoadVideo</sender>
   <signal>clicked()</signal>
   <receiver>VideoLoadDialog</receiver>
   <slot>LoadVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveVideo</sender>
   <signal>clicked()</signal>
   <receiver>VideoLoadDialog</receiver>
   <slot>SaveVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LoadVideo()</slot>
  <slot>SaveVideo()</slot>
 </slots>
</ui>
